src/ngraph/frontend/onnx_import/op/round.cpp

```diff
@@ -0,0 +1,42 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "round.hpp"
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include "default_opset.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector round(const Node& node)
+                {
+                    const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
+                    return {std::make_shared<ngraph::op::Round>(data)};
```

Comment:
In current convention it should be `ngraph::opset0::Round`

Reply:
Done